src/main/java/frc/robot/subsystems/scoring/AimerIOSim.java

```diff
@@ -9,7 +9,7 @@
 
 public class AimerIOSim implements AimerIO {
     // TODO: Tune this later
-    private final SingleJointedArmSim sim = new SingleJointedArmSim(DCMotor.getNeoVortex(2), 1.0, 0.0, 1.0, 0.0, 1.0, true, 0.0);
+    private final SingleJointedArmSim sim = new SingleJointedArmSim(DCMotor.getNeoVortex(2), 1.0, 1.0, 1.0, 0.0, 1.0, true, 0.0);
```

Comment:
what's this doing in this PR 👀

Reply:
I think that it got added when I rebased

Reply:
Oh... No, it's because the 0 was crashing thee code.